src/validation/validation.hpp

```diff
@@ -123,6 +125,11 @@ void create_synthetic_rsf_tab(View5D &rsf_tab, const int nw, const int nump,
                               const int numsza, const int numcolo3,
                               const int numalb, Real *synthetic_values);
 
+void convert_1d_std_to_2d_view_device(const std::vector<Real> &pmid_db,
```

Comment:
Can you put comments in front of these functions that indicate they're copying to and from vectors? I know it's pretty obvious, but the `const` in front of `var_device` in this function might confuse some people. Also, consider replacing `std` with `vector` (though I think it's fine to keep it as `std` if that's what you decide).

Reply:
Done.